CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ if(BUILD_4 AND BUILD_D)
   set(kinds "4" "d")
 elseif(BUILD_4 AND NOT BUILD_D)
   set(kinds "4")
+  if(BUILD_TESTING)
```

Comment:
What is this all about?

Reply:
If BUILD_TESTING=ON but we're only building _4, the cmake will fail because the tests currently only work for _d, so it will give a bunch of errors related to the _d version of the library not being available. The plan will be to revert this in the next version when we add testing for the _4 case, but for now that ended up being more difficult than I hoped.

Reply:
OK, is there an issue that describes what you plan to do?

Reply:
#162